test/export/test_sparse.py

```diff
@@ -0,0 +1,229 @@
+    )
+    @parametrize("dtype", DTYPES)
+    @parametrize("itype", ITYPES)
+    @all_sparse_layouts("layout")
+    def test_sumnet(self, dtype, itype, layout):
+        if layout is not torch.sparse_coo:
+            self.skipTest("TODO: support non-coo sparsity!")
+
+        net = SumNet()
+        for sparse_input in self.generate_simple_inputs(
+            layout,
+            device="cpu",
+            dtype=dtype,
+            index_dtype=itype,
+        ):
+            # Build the traced graph.
+            prog = torch.export.export(net, (sparse_input,))
+            # Test arg/sum/output.
+            for i, node in enumerate(prog.graph.nodes):
+                meta = node.meta.get("val", None)
+                if i == 0:
+                    self.assertEqualMeta(meta, sparse_input)
+                elif i == 1:
+                    self.assertIsInstance(meta, FakeTensor)
+                    self.assertEqual(meta.layout, torch.strided)
+                    self.assertEqual(meta.dtype, dtype)
+                else:
+                    self.assertEqual(meta, None)
+
+    @unittest.skipIf(
+        sys.version_info >= (3, 12), "torch.compile is not supported on python 3.12+"
+    )
+    @parametrize("dtype", DTYPES)
+    @parametrize("itype", ITYPES)
+    @all_sparse_layouts("layout")
+    def test_eltwisenet(self, dtype, itype, layout):
+        if layout is not torch.sparse_coo:
+            self.skipTest("TODO: support non-coo sparsity!")
+
+        net = EltwiseNet()
+        for sparse_input in self.generate_simple_inputs(
+            layout,
+            device="cpu",
+            dtype=dtype,
+            index_dtype=itype,
+        ):
+            # Build the traced graph.
+            prog = torch.export.export(net, (sparse_input,))
+            # Test arg/neg/abs/mul/relu/output.
+            for i, node in enumerate(prog.graph.nodes):
+                meta = node.meta.get("val", None)
+                if i <= 4:
+                    self.assertEqualMeta(meta, sparse_input)
+                else:
+                    self.assertEqual(meta, None)
+
+    @unittest.skipIf(
+        sys.version_info >= (3, 12), "torch.compile is not supported on python 3.12+"
+    )
+    def test_activation_coo(self):
+        net = SparseActivationCOO()
+        x = [torch.randn(3, 3) for _ in range(3)]
+        # Build the traced graph.
+        prog = torch.export.export(net, args=(x,))
+        # Test args/to_sparse/output.
+        for i, node in enumerate(prog.graph.nodes):
+            meta = node.meta.get("val", None)
+            if i <= 2:
+                self.assertIsInstance(meta, FakeTensor)
+                self.assertEqual(meta.layout, torch.strided)
+                self.assertEqual(meta.dtype, x[i].dtype)
+            elif i <= 5:
+                self.assertEqualMeta(meta, x[i - 3].to_sparse())
+            else:
+                self.assertEqual(meta, None)
```

Comment:
Similarly to above:
```suggestion
            if i <= 2:
                self.assertIsInstance(meta, FakeTensor)
                self.assertEqualMeta(meta, x[i])
            elif i <= 5:
                self.assertEqualMeta(meta, net(x)[i - 3])
            else:
                self.assertEqual(meta, None)
```

`net(x)[i - 3]` looks better than `x[i - 3].to_sparse()`, imho :)